recipe/meta.yaml

```
@@ -118,6 +118,7 @@ outputs:
         # 2022/03/29 hmaarrfk
         # It seems that this test can cause segmentation faults on the CIs.
         {% set tests_to_skip = tests_to_skip + " or test_write_video_with_audio" %}
+        {% set tests_to_skip = tests_to_skip + " or test_video_clips_custom_fps" %} 
```

Let's not skip aarch-specific tests globally please (also below)
```suggestion
        {% set tests_to_skip = tests_to_skip + " or test_video_clips_custom_fps" %}  # [aarch64]`
```

---

This test actually failed on Linux-64 .. not on aarch

---

See https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=644049&view=logs&jobId=9a864fd9-6c8f-52ca-79ce-2aa6dca1a1de

---

Interesting. Why didn't they fail before then? I see no changed host dependency based on the pinnings here.

---

I guess it’s just a sporadic failure

---

Launched another run without the skip, let's see what happens.

---

See https://github.com/conda-forge/torchvision-feedstock/pull/64#issuecomment-1396753883 - resolving here.

---

I think their av integration is buggy.

---

Agreed. Are we happy to merge then @hmaarrfk @h-vetinari?